src/sentry/models/event.py

```diff
@@ -320,47 +310,205 @@ def site(self):
 
     @property
     def server_name(self):
-        warnings.warn('Event.server_name is deprecated. Use Event.tags instead.')
+        warnings.warn(
+            'Event.server_name is deprecated. Use Event.tags instead.',
+            DeprecationWarning)
         return self.get_tag('server_name')
 
     @property
     def checksum(self):
         warnings.warn('Event.checksum is no longer used', DeprecationWarning)
         return ''
 
-    def error(self):
+    def error(self):  # TODO why is this not a property?
```

Comment:
Can we just start killing these? Nothing should use this any more after the last changes where we introduced the title but I don't know if I caught all references (plugins -.-).

Reply:
is there any way to even tell what plugins are using this? I mean there are core plugins, but are there others that we only know about once we break them?

Reply:
Catching plugins that are not part of sentry.io is really hard as they could be literally anywhere. If it isn't installed as part of sentry.io I wouldn't worry about it as it is likely in a repository we don't manage and likely don't know about.

Reply:
I can delete this in a later PR, I don't want to add another unrelated weird reason to have to back out of this one